content/docs/troubleshooting/logging.md

```diff
@@ -0,0 +1,43 @@
+# Centralized logging framework of Tinkerbell
+
+## Overview
+
+Tinkerbell as of today is a composition of following microservices
+- tink-server
+- hegel
+- boots
+- db
+- cli
+- nginx
+- registry
+- tink-worker
+- action containers
+
+
+Logging is crucial for any product or feature to ensure debugging and bug-free implementation. Logs of these services can be obtained using `docker logs <service name>`. For tink-worker and action containers, the steps are same and many a times it would happend that service containers has exited once the execution is completed leaving the user with an overhead of first searching the dead container `docker ps -a` to check the logs of that container. Also, logs are segregated across the nodes. So, the user will have to jump across the nodes to check the logs of the respective container or microservice. With the advent of more feature on these services microservices `docker logs` will become laborious to use from the console.
+
+Centralized logging implementation tries to solve this problem. This feature adds the support of
+- Log aggregation of these microservices to a log server.
+- Logs redirection to a file thus NO dependency on `docker logs`. If the user wants he can create a logging container and mount log directory.
+- Log rotation is added for the file.
+- Plug and Play. (Please check implementation details)
+
+## Implementation details
+
+Centralized logging only captures docker or microservices logs. It leverages the [`logging driver`](https://docs.docker.com/config/containers/logging/configure/) and `logging option` parameter of the docker cli. Currently, we are supporting `server_address` and `tag` log-opts but in next releases, we will support other log-opts as well.
```

Comment:
You can centralize logs configuring the docker daemon with the right logging driver.